packages/updatePrescriptionStatus/src/updatePrescriptionStatus.ts

```diff
@@ -0,0 +1,217 @@
+
+interface DynamoDBItem {
+  RequestID: string | undefined;
+  PrescriptionID: string;
+  PatientNHSNumber: string;
+  PharmacyODSCode: string;
+  TaskID: string;
+  LineItemID: string;
+  TerminalStatusIndicator: string;
+  LastModified: string;
+  Timestamp: string;
+  RequestMessage: any;
+}
+
+const lambdaHandler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  const xRequestId = event.headers["x-request-id"]
+  let requestBody
+  try {
+    requestBody = JSON.parse(event.body || "")
+  } catch (jsonParseError) {
+    logger.error("Error parsing JSON", {error: jsonParseError})
+    const errorResponseBody = {
+      resourceType: "OperationOutcome",
+      issue: [
+        {
+          code: "value",
+          severity: "error",
+          details: {
+            coding: [
+              {
+                system: "https://fhir.nhs.uk/CodeSystem/http-error-codes",
+                code: "BAD_REQUEST",
+                display: "400: The Server was unable to process the request."
+              }
+            ]
+          }
+        }
+      ]
+    }
+    return {
+      statusCode: 400,
+      body: JSON.stringify(errorResponseBody),
+      headers: {
+        "Content-Type": "application/fhir+json",
+        "Cache-Control": "no-cache"
+      }
+    }
+  }
+
+  const entries = requestBody.entry
+
+  if (!entries || entries.length === 0) {
+    logger.error("Missing required fields")
+    const errorResponseBody = {
+      resourceType: "OperationOutcome",
+      issue: [
+        {
+          code: "value",
+          severity: "error",
+          details: {
+            coding: [
+              {
+                system: "https://fhir.nhs.uk/CodeSystem/Spine-ErrorOrWarningCode",
+                code: "MISSING_VALUE",
+                display: "Missing required fields"
+              }
+            ]
+          }
+        }
+      ]
+    }
+    return {
+      statusCode: 400,
+      body: JSON.stringify(errorResponseBody),
+      headers: {
+        "Content-Type": "application/fhir+json",
+        "Cache-Control": "no-cache"
+      }
+    }
+  }
+
+  const responseBundle: any = {
+    resourceType: "Bundle",
+    id: uuidv4(),
+    meta: {
+      lastUpdated: new Date().toISOString()
+    },
+    type: "transaction-response",
+    entry: []
+  }
+
+  for (const entry of entries) {
+    logger.info("Processing entry", {entry: entry})
+    const entry_resource = entry.resource
+
+    const dynamoDBItem: DynamoDBItem = {
+      RequestID: xRequestId,
+      PrescriptionID: entry_resource.basedOn?.[0]?.identifier?.value,
+      PatientNHSNumber: entry_resource.for?.identifier?.value,
+      PharmacyODSCode: entry_resource.owner?.identifier?.value,
+      TaskID: entry_resource.id,
+      LineItemID: entry_resource.focus?.identifier?.value,
+      TerminalStatusIndicator: entry_resource.status,
+      LastModified: entry_resource.lastModified,
+      Timestamp: new Date().toISOString(),
```

Comment:
I don't think we need the current timestamp as well as the lastmodified

Reply:
The `Timestamp` and `LastModified` fields have been removed.